ufc/flags-v1-obfuscated.json

```diff
@@ -64,6 +64,62 @@
       ],
       "totalShards": 10000
     },
+    "3d88068611498391d5c8307467e6fa2c": {
```

Comment:
Note for reviewers: this file is auto-generated based on the unobfuscated version -- please just review that one